app-misc/ckb/ckb-0.5.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+DESCRIPTION="Corsair K65/K70/K95 Driver"
+HOMEPAGE="https://github.com/ckb-next/ckb-next"
+SRC_URI="https://github.com/ckb-next/ckb-next/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="systemd"
+
+RDEPEND="
+	dev-libs/libdbusmenu-qt
+	dev-libs/quazip:0=
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	media-sound/pulseaudio
+	virtual/libudev:=
+	x11-libs/libX11
+	x11-libs/libxcb:=
+	x11-libs/xcb-util-wm"
+DEPEND="${RDEPEND}"
+BDEPEND="dev-qt/linguist-tools:5"
+
+S="${WORKDIR}/${PN}-next-${PV}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DDISABLE_UPDATER=yes
+		-DFORCE_INIT_SYSTEM=$(usex systemd systemd openrc)
+	)
+	cmake_src_configure
+}
+
+src_install() {
+	cmake_src_install
+	dodoc CHANGELOG.md
+
+	newinitd "${FILESDIR}"/ckb.initd ckb-daemon
+}
+
+pkg_postinst() {
```

Comment:
```
 * QA Notice: package is installing udev rules without inheriting udev.eclass!
 * Packages must inherit udev.eclass then call udev_reload in pkg_postinst.
```

Can you add the appropriate call here? Will need to `inherit udev` too.

Reply:
Oh, this is https://bugs.gentoo.org/854747. Once the call is added, you can add a `Closes` tag to the commit for this bug too.